### PR TITLE
VM table outputs

### PR DESCRIPTION
**Pull request by derekbekoe**
This PR focuses on the get/list/show commands.

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py [hunk 1]
```diff
@@ -45,23 +45,24 @@
...
-cli_command('vm list-vm-resize-options', VirtualMachinesOperations.list_available_sizes, factory)
-cli_command('vm stop', VirtualMachinesOperations.power_off, factory)
-cli_command('vm restart', VirtualMachinesOperations.restart, factory)
-cli_command('vm start', VirtualMachinesOperations.start, factory)
-cli_command('vm redeploy', VirtualMachinesOperations.redeploy, factory)
-cli_command('vm list-ip-addresses', list_ip_addresses)
-cli_command('vm list', list_vm, simple_output_query="[*].{Name: name, ResourceGroup: resourceGroup, Location: location, VmSize: hardwareProfile.vmSize, Urn: join(':', [storageProfile.imageReference.publisher, storageProfile.imageReference.offer, storageProfile.imageReference.sku, storageProfile.imageReference.version])} | sort_by(@, &Name)")
-cli_command('vm resize', resize_vm)
-cli_command('vm capture', capture_vm)
+cli_command('vm delete', VirtualMachinesOperations.delete, factory) # no output
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py]: you probably don't this comment ` # no output`, as it is not being applied on all cases

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py [hunk 2]
```diff
@@ -112,50 +113,50 @@
...
+cli_command('vm image list', list_vm_images, simple_output_query="[*].{Publisher:publisher, Offer:offer, Sku:sku, Version:version, UrnAlias:\"urn alias\", Urn:urn}")
 
 # VM Usage
 factory = lambda _: _compute_client_factory().usage
-cli_command('vm usage list', UsageOperations.list, factory)
+cli_command('vm usage list', UsageOperations.list, factory, simple_output_query="[*].{Name:name.localizedValue, CurrentValue:currentValue, Limit:limit} | sort_by(@, &Name)")
 
 # VM ScaleSet
 factory = lambda _: get_mgmt_service_client(VMSSClient).vmss
 cli_command('vmss create', VmssOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting vmss create'))
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py]: Do you want to chat with Burt to output some when the creating is done?

**Comment by yugangw-msft** [comment]
LGTM. Feel free to merge once my comments are clarified.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py (hunk 1 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py]: Yes these should have been removed beforehand.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py (hunk 2 above)
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py]: There's no output from vmss create so no need for table output.

**Pull request merged**